src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import java.util.Scanner;
+import java.lang.String;
+
 public class Duke {
+    static final int MAX_NO = 100;
+    private static int numOfTasks = 0; // stores the number of tasks in the array
+    private static Task[] tasks = new Task[MAX_NO]; // initialise array of Task objects
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        getMessage();
+    }
+
+    /**
+     * gets command(s) from the user and executes it(them) appropriately
+     * available commands: list, done, deadline, event, todo, task, bye
+     */
+    private static void getMessage() {
+        Scanner in = new Scanner(System.in);
+        String[] input = new String[MAX_NO];
+        int i = 0;
```

Comment:
May be better to indicate what does i stands for.

Reply:
okay i'll add a comment in the code :D